packages/deploy-helpers/bases/deploy-apm.js

```diff
@@ -0,0 +1,141 @@
+const namehash = require('eth-ens-namehash').hash
```

Comment:
Do you think it makes sense to have scripts to deploy the bases in this package?